### PR TITLE
Added the docker script

### PR DESCRIPTION
**Pull request by Pchelolo**
A bit of code duplication, but it works.. 

cc @wikimedia/services

### CODE REVIEW

**Comment by Pchelolo** [comment]
The `docker.js` is obviously a direct copy-paste from the service-runner. The `index.js` contains parameter parsing plus some copy-paste code to get the config/app base path (for docker-run and docker-test) commands.

File: index.js [hunk 1]
```diff
@@ -1,3 +1,165 @@
...
+            nargs: 1,
+            global: true
+        },
+        verbose: {
+            default: false,
+            describe: 'be verbose',
+            type: 'boolean',
+            global: true
+        },
+        v: {
```
Comment (d00rman) [inline index.js]: Do we need this at all?

File: index.js [hunk 2]
```diff
@@ -1,3 +1,165 @@
...
+            return defValue;
+        }
+        return '';
+    });
+}
+
+function main() {
+    const options = getOptions();
+    let configFile = options.configFile;
+    if (!/^\//.test(configFile)) {
```
Comment (d00rman) [inline index.js:148]: Only docker-start really needs the config, the rest of them don't so I would prefer if we could eliminate the need for having to supply a dummy config.

File: index.js [hunk 3]
```diff
@@ -1,3 +1,165 @@
...
+        },
+        v: {
+            alias: 'version',
+            default: false,
+            describe: 'print the service\'s version and exit',
+            type: 'boolean',
+            global: true
+        }
+    })
+    .command('docker-start', 'starts the service in a Docker container')
```
Comment (d00rman) [inline index.js]: Perhaps change it into `run` or something similar? The `docker-` prefix seems a bit superfluous now.

File: index.js [hunk 4]
```diff
@@ -1,3 +1,165 @@
...
+    .command('docker-test', 'starts the test process in a Docker container')
```
Comment (d00rman) [inline index.js]: Idem

**Review by d00rman — CHANGES_REQUESTED** [review]
LGTM in general, some comments in-lined

File: index.js (hunk 2 above)
Comment (Pchelolo) [reply index.js:148]: @d00rman Yup, this is just the first iteration - here I wanted to just copy-paste as much as possible and then start refactoring in the next PR. There's quite a few other things I want to change here

Comment (d00rman) [reply index.js:148]: Makes sense

File: package.json [hunk 5]
```diff
@@ -27,6 +27,12 @@
   "engines": {
     "node": ">=4.2.2"
   },
+  "dependencies": {
```
Comment (d00rman) [inline package.json:30]: On line 7, it says the bin's name should be `service-runner`. Obviously, it needs `s/runner/builder`

**Review by d00rman — CHANGES_REQUESTED** [review]
One other minor thing before merging

**Review by d00rman — APPROVED** [review]

**Comment by Pchelolo** [comment]
Gracias. No the next logical step would be to remove this code from `service-runner` and then start tweaking it in this repository.

**Comment by d00rman** [comment]
We can do it independently, really

**Pull request merged**